lib/bundler/rubygems_gem_installer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "rubygems/installer"
+require "digest/sha2"
```

Comment:
instead of this, we should use `Digest(:SHA256)` on line 52?

Reply:
Yea, that does look better to use and it's theadsafe as well which i think is the the problem we're dealing with.  https://github.com/ruby/ruby/blob/trunk/ext/digest/lib/digest.rb#L96

Reply:
ping on making this change?

Reply:
updated, sorry for the delay 🙇